commit 99f504ba70da2018c32504396fff96bc3295c964 (refs/changes/60/3960/3)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-05-16T14:46:44-04:00 (1 year, 5 months ago)
    
    TRITON-388 napi crashes when first starting
    Reviewed by: Cody Peter Mello <melloc@writev.io>
    Approved by: Cody Peter Mello <melloc@writev.io>

diff --git a/lib/napi.js b/lib/napi.js
index 3f6ae21..4eb9b8f 100644
--- a/lib/napi.js
+++ b/lib/napi.js
@@ -295,9 +295,9 @@ NAPI.prototype.state_init.buckets = function (S) {
     models.init(self, function (err) {
         if (err) {
             self.log.error(err, 'Error initializing models; retrying in 10s');
-            S.timeout(function () {
+            S.timeout(10000, function () {
                 S.gotoState('init.buckets');
-            }, 10000);
+            });
             return;
         }
 
